tokens/contracts/erc-1155/ERC1155Lazy.sol

```diff
@@ -60,7 +59,6 @@ abstract contract ERC1155Lazy is IERC1155LazyMint, ERC1155BaseURI, Mint1155Valid
         address sender = _msgSender();
 
         require(minter == sender || isApprovedForAll(minter, sender), "ERC1155: transfer caller is not approved");
-        require(isValidMinter(minter), "ERC1155: minter not granted");
```

Comment:
Just to confirm, is this ok to remove that as the function is still public?
It's being overridden by the parent contract, but I just want to make sure that's ok.

Reply:
`isValidMinter()` is a call we added and points to `MinterAccessControl` which is not available anymore in this class.